packages/fhir-converter/src/templates/cda/Resources/RelatedPerson.hbs

```diff
@@ -29,7 +29,7 @@
                 },
                 {
                     {{#with (evaluate 'Utils/ContactRoleCode.hbs' code=(trimAndUpper ../relatedPerson.classCode)) as |respObjRole|}}
-                        "text":"{{{respObjRole.code.displayName}}}",
```

Comment:
we actually control the output here so this is not dangerous but no reason to have it